tests/core/cgo/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    cgo = True,
+    copts = ["-DRULES_GO_C"],
+    cxxopts = ["-DRULES_GO_CPP"],
+    srcs = [
```

Comment:
Define something in `cppopts` and check that define is visible in both add.c and add.cpp.

Reply:
👍